tests/unit/python/foglamp/common/microservice_management_client/test_microservice_management_client.py

```diff
@@ -267,3 +267,224 @@ def test_get_services_client_err(self, status_code):
             with patch.object(HTTPConnection, 'getresponse', return_value=response_mock) as response_patch:
                 with pytest.raises(client_exceptions.MicroserviceManagementClientError) as excinfo:
                     ret_value = ms_mgt_client.get_services('foo', 'bar')
+
+    def test_get_configuration_category(self):
+        microservice_management_host = 'host1'
+        microservice_management_port = 1
+        ms_mgt_client = MicroserviceManagementClient(
+            microservice_management_host, microservice_management_port)
+        response_mock = MagicMock(type=HTTPResponse)
+        undecoded_data_mock = MagicMock()
+        response_mock.read.return_value = undecoded_data_mock
```

Comment:
Better to make a fixture and use it , as above lines are repeated for every test.